xls2xml/amp-t2d/T2D_xls2xml.xslt

```diff
@@ -0,0 +1,151 @@
+        <xsl:value-of select="Center_name"/>
+      </xsl:attribute>
+      <TITLE><xsl:value-of select="Title"/></TITLE>
+      <STUDY_REF>
+        <xsl:attribute name="refname">
+          <xsl:value-of select="Project_name"/>
+        </xsl:attribute>
+      </STUDY_REF>
+    </EXPERIMENT>
+  </EXPERITMENT_SET>
+</xsl:template-->
+
+<!--> Sample <-->
+<xsl:template match="/SampleSet"><!-->Should match <key_in_config>+'Set'<-->
+  <SAMPLE_SET noNamespaceSchemaLocation="ftp://ftp.sra.ebi.ac.uk/meta/xsd/sra_1_5/SRA.sample.xsd">
+    <xsl:for-each select="Sample"><!-->Should select from <key_in_config><-->
+      <SAMPLE>
+        <xsl:attribute name="alias">
+          <xsl:value-of select="Sample_ID"/>
+        </xsl:attribute>
+        <xsl:attribute name="center_name">
+          <xsl:value-of select="Center_name"/>
+        </xsl:attribute>
+        <DESCRIPTION><xsl:value-of select="Description"/></DESCRIPTION>
+        <SAMPLE_NAME display_name="Human">
+          <TAXON_ID>9606</TAXON_ID>
+          <SCIENTIFIC_NAME>homo sapiens</SCIENTIFIC_NAME>
+          <COMMON_NAME>human</COMMON_NAME>
+        </SAMPLE_NAME>
+        <SAMPLE_ATTRIBUTES>
+			    <SAMPLE_ATTRIBUTE>
+            <TAG>subject_id</TAG>
+            <VALUE><xsl:value-of select="Subject_ID"/></VALUE>
+          </SAMPLE_ATTRIBUTE>
+          <SAMPLE_ATTRIBUTE>
+            <TAG>phenotype</TAG>
+            <VALUE><xsl:value-of select="Phenotype"/></VALUE>
+          </SAMPLE_ATTRIBUTE>
+          <SAMPLE_ATTRIBUTE>
+            <TAG>cell_type</TAG>
+            <VALUE><xsl:value-of select="Cell_Type"/></VALUE>
+          </SAMPLE_ATTRIBUTE>
+          <SAMPLE_ATTRIBUTE>
+            <TAG>gender</TAG>
+            <VALUE><xsl:value-of select="Gender"/></VALUE>
+          </SAMPLE_ATTRIBUTE>
+          <SAMPLE_ATTRIBUTE>
+            <TAG>T2D</TAG>
+            <VALUE><xsl:value-of select="T2D"/></VALUE>
+          </SAMPLE_ATTRIBUTE>
+          <SAMPLE_ATTRIBUTE>
+            <TAG>year_of_birth</TAG>
+            <VALUE><xsl:value-of select="Year_of_Birth"/></VALUE>
+          </SAMPLE_ATTRIBUTE>
+        </SAMPLE_ATTRIBUTES>
+      </SAMPLE>
+    </xsl:for-each>
+  </SAMPLE_SET>
+</xsl:template>
+
+<!--> Study <-->
+<xsl:template match="/ProjectSet">
+  <STUDY_SET noNamespaceSchemaLocation="ftp://ftp.sra.ebi.ac.uk/meta/xsd/sra_1_5/SRA.study.xsd">
+    <xsl:for-each select="Project">
+      <STUDY>
+        <xsl:attribute name="alias">
+          <xsl:value-of select="Project_Accronym"/>
+        </xsl:attribute>
+        <DESCRIPTOR>
+          <STUDY_TITLE><xsl:value-of select="Project_Name"/></STUDY_TITLE>
+          <STUDY_ABSTRACT><xsl:value-of select="Project_Description"/></STUDY_ABSTRACT>
+        </DESCRIPTOR>
+      </STUDY>
+      <STUDY_ATTRIBUTES>
+        <STUDY_ATTRIBUTE>
+          <TAG>PI(s)</TAG>
+          <VALUE><xsl:value-of select="PI_s_"/></VALUE>
+        </STUDY_ATTRIBUTE>
+      </STUDY_ATTRIBUTES>
+    </xsl:for-each>
```

Comment:
We should add the Project_website link, External Links and Data URL as a LINK_TYPE in the study xml. Also project Publication as a link to PUBMED.